### PR TITLE
AMW-268 Fix ansible-common-criteria-demo on Olympus.

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/AMW-268

Olympus (with using sudo: yes) [ansible-common-criteria-demo]

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@rpelisse Can you please review the PR?

File: molecule/tomcat/prepare.yml [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
 
     - name: Install required dependencies
       package:
-        name: sudo
\ No newline at end of file
+        name: sudo
+
+  post_tasks:
+    - name: "Download latest Apache Tomcat zip file from {{ download_url }}."
+      get_url:
```
Comment (rpelisse) [inline molecule/tomcat/prepare.yml]: @RanabirChakraborty tiny detail, but can you add the missing ansible.builtin. in front of each tasks (here, in front of get_url, but also elsewhere in the file)

File: molecule/wildfly/molecule.yml [hunk 2]
```diff
@@ -4,29 +4,21 @@ dependency:
...
-  config_options:
-    defaults:
-      interpreter_python: auto_silent
-    ssh_connection:
-      pipelining: false
   playbooks:
+    converge: converge.yml
     prepare: prepare.yml
-    converge: ../../playbooks/playbook_wildfly.yml
+    verify: ../../playbooks/playbook_wildfly.yml
```
Comment (rpelisse) [inline molecule/wildfly/molecule.yml]: @RanabirChakraborty let's rename those playbooks wildfly.yml and tomcat.yml. Having playbook_ in front of their name inside a folder called playbooks is a bit redundant...

File: playbooks/playbook_tomcat.yml [hunk 3]
```diff
@@ -1,41 +1,12 @@
...
-  become_user: root
   vars_files:
-    - vars.yml
-  collections:
-    - middleware_automation.jws
-    - middleware_automation.common_criteria
-  roles:
-    - jws
-    - jws_common_criteria
+    - ../molecule/tomcat/vars.yml
```
Comment (rpelisse) [inline playbooks/playbook_tomcat.yml]: It's a bit weird to have the "demo" playbook import a set of vars from the testsuite. I would suggest to create a vars/ folder inside playbooks/ and move this file there (so playbooks/vars/tomcat.yml and same for wildfly.yml).

**Review by rpelisse — CHANGES_REQUESTED** [review]
A few little enhancements suggested, but overall looks good to me, good work!

File: playbooks/playbook_tomcat.yml (hunk 3 above)
Comment (RanabirChakraborty) [reply playbooks/playbook_tomcat.yml]: Thanks! done

File: molecule/wildfly/molecule.yml (hunk 2 above)
Comment (RanabirChakraborty) [reply molecule/wildfly/molecule.yml]: Thanks! done

File: molecule/tomcat/prepare.yml (hunk 1 above)
Comment (RanabirChakraborty) [reply molecule/tomcat/prepare.yml]: Thanks! done , pleas review once.

**Review by rpelisse — APPROVED** [review]

**Pull request merged**